src/MessagingThreadHistory/MessageMetadata.tsx

```diff
@@ -12,21 +12,25 @@ interface Props {
   className?: string;
   placement: "left" | "right" | "center";
   timestamp?: Date;
+  showReadStatus?: boolean;
   children: React.ReactNode;
 }
 
 export const MessageMetadata: React.FC<
   Props & { ref?: React.Ref<HTMLDivElement> }
 > = React.forwardRef((props: Props, ref: React.Ref<HTMLDivElement>) => {
-  const { className, placement, timestamp, children } = props;
+  const { className, placement, timestamp, showReadStatus, children } = props;
   return (
-    <div ref={ref} className={classNames(cssClass(`Message--${placement}`), className)}>
-      {children}
-      {timestamp && (
-        <span className={cssClass(`Timestamp--${placement}`)}>
-          {_formatDateForTimestamp(timestamp)}
-        </span>
-      )}
+    <div className={cssClass("Message--container")}>
+      <div ref={ref} className={classNames(cssClass(`Message--${placement}`), className)}>
+        {children}
+        {timestamp && (
+          <span className={cssClass(`Timestamp--${placement}`)}>
+            {_formatDateForTimestamp(timestamp)}
+          </span>
+        )}
+      </div>
+      {showReadStatus && <div className={cssClass("ReadReceipt")}>Read</div>}
```

Comment:
I think we're going to need some way for a consumer to specify the text to use here. Currently, we're able to translate the text in messaging components in the FamPo by passing the translation through as a prop. Ideally, we'd like to switch to translating in `clever-components` so that isn't necessary, but we're not at that point yet.

Reply:
Just changed it to a consumer passed text field! Thanks for the reminder!

Reply:
not related to this PR, but chiming in that `switch to translating in clever-components` may not be the move since `clever-components` is a public repo and translation would seem to depend on Clever user data, no?